eng/pipelines/templates/steps/configure-oidc-auth.yml

```diff
@@ -0,0 +1,16 @@
+parameters:
+  ServiceConnectionName: ""
```

Comment:
The default value should be `azd-service-connection` here so you don't have to duplicate that into the other templates